pages/tech/environments.md

```diff
@@ -0,0 +1,92 @@
+import Section from '../../components/section'
+
+# Computational Environments
+
+## Overview
+
+At CarbonPlan, we use different types of computational environments to support our data science work:
+
+- **Local environments**: For day-to-day development
+- **Docker containers**: For reproducibility and deployment
+
+These environments support various activities across our workflow:
+
+| Activity        | Local Environment   | Docker                   |
+| --------------- | ------------------- | ------------------------ |
+| Development     | ✅ Primary choice   | ⚠️ Can be slower         |
+| Testing         | ✅ Quick iterations | ✅ CI integration        |
+| Deployment      | ❌ Not recommended  | ✅ Best practice         |
+| Reproducibility | ⚠️ Limited          | ✅ Excellent             |
+| Collaboration   | ⚠️ Setup required   | ✅ Consistent experience |
```

Comment:
@katamartin, do we support rendering markdown tables? this isn't getting rendered and i'm wondering if i'm missing something

Reply:
We don't currently 😕 . For production purposes, we almost always need responsive styles, and we use the [`Table` component](https://carbonplan.org/design/table) for that.